### DISCUSSION TITLE
Ability for players to choose main respawn home (or maybe other options too)

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#4338] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Hi, could you please add way for players that they can choose from their home, which will be the main one where they should be respawn? So if player has 5 homes, he can choose which one of these 5 homes will be the main respawn one. Maybe some option with shift for ex. to highligh it, or some other button in /homes gui, or maybe special command to allow them choose which home... and maybe also let them allow other options, for ex. if they have homes, but they wanna be rather respawned on spawn, let them allow it.
### Basic explanation as to why this feature is needed and/or useful to have: ###
Because now players can't decide on which home they will respawn. It can be better for them, if they will respawn there, where they wanna, and not for ex. in mines, where they will have home too.

**Comment by CloudeLecaw** [comment]
x. if they have homes, but they wanna be rather respawned on spawn, let them allow it.

its the admins or owners work to do this. its not good if one can spawn at spawn (spawn is not a home)
or other crazzy stuff. a home normaly is not a spawnpoint.
better would be allow them homes and define a spawn.

**Comment by bobhenl** [comment]
My priority order is this:
```
    PriorityOrder:
    - homeLocation
    - bedLocation
    - spawn
    - worldSpawn
```
So the main is home, but they can't choose which one for now.

**Comment by CloudeLecaw** [comment]
used is players default home if exists
if players default home does not exists thenplayers bed location
if if players bed location does not exists then CMI spawn
if cmi spawn is not set then vanillas worldspawn (minecraft:setworldspawn)

**Comment by bobhenl** [comment]
Yeah, I'm just suggesting that players could choose their default (main) home. If they have many named homes, it will be worthy.

**Comment by CloudeLecaw** [comment]
A good idea :)
